sqlite/src/main/java/org/jdbi/v3/sqlite3/URLColumnMapper.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jdbi.v3.sqlite3;
+
+import org.jdbi.v3.core.mapper.ColumnMapper;
+import org.jdbi.v3.core.statement.StatementContext;
+
+import java.net.MalformedURLException;
+import java.net.URI;
+import java.net.URL;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
+public class URLColumnMapper implements ColumnMapper<URL> {
+
+    @Override
+    public URL map(ResultSet r, int columnNumber, StatementContext ctx) throws SQLException {
+        URL url = null;
+
+        try {
+            url = URI.create(r.getString(columnNumber)).toURL();
+        } catch (MalformedURLException e) {
+            e.printStackTrace();
```

Comment:
I'm of two minds on this exception. On one hand it seems like the more correct approach is to raise the exception rather than swallow it. On the other hand, that means if a malformed URL somehow gets into the database, you could never read that row out again without an exception--lame.

Any any rate, this should be logged with SLF4J rather than printed to STDOUT

Reply:
I think I'd rather it throw.  The data you requested from the database cannot be represented in the requested container type.  `null` is indistinguishable from a valid return of a different value, so that's no good.  So we must fail.

If your data may be a URL or maybe not, you should fetch it as a string and guard the conversion, IMO.  Or we could make this a configurable policy with choices of `THROW` vs `SWALLOW_RETURN_NULL`.  But I think the default has to be to throw the exception.

Reply:
Yes, I think you're right. +1 to throw